hw/ip/csrng/doc/registers.md

```diff
@@ -157,30 +157,51 @@ Command request register
 ## SW_CMD_STS
 Application interface command status register
 - Offset: `0x1c`
-- Reset default: `0x1`
-- Reset mask: `0x3`
+- Reset default: `0x0`
```

Comment:
I'd also combine this commit with the first one because otherwise documentation will be outdated